samples/highcharts/blog/chart-inside-tooltip/demo.js

```diff
@@ -0,0 +1,51 @@
+function renderChart(point) {
+    Highcharts.chart('hc-tooltip', {
+        chart: {
+            type: 'pie'
+        },
+        title: {
+            text: null
+        },
+        subtitle: {
+            text: 'Chart inside tooltip'
+        },
```

Comment:
Why a null title and an active subtitle? If it is for styling reasons, it would be more semantically correct to set a smaller font size for the title and then skip the subtitle.